src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -101,16 +123,55 @@ export default class VizShonenJump extends Connector {
             });
     }
 
+    async _getMangaVolumes(manga) {
+        const request = new Request(new URL(manga.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request);
+
+        if ( data.innerText.includes('Log in to view your library') ) {
+            alert('Lost your login cookie between fetching the manga list and now (fetching the volume list).', this.label, 'info');
```

Comment:
Just throw and let HakuNeko do the error handling `throw new Error('Message whats wrong and what a user needs to do ...')`